lib/src/configuration.dart

```diff
@@ -138,6 +141,7 @@ class Configuration {
     waitForProcessing = configValues.waitForProcessing ?? false;
     authToken = configValues.authToken; // or env. var. SENTRY_AUTH_TOKEN
     url = configValues.url; // or env. var. SENTRY_URL
+    urlPrefix = configValues.urlPrefix; // or env. var. SENTRY_URL_PREFIX
```

Comment:
did you test if `SENTRY_URL_PREFIX` works? because I can't find it in the sentry-cli configuration: https://docs.sentry.io/product/cli/configuration/

Reply:
and also please add a `README.md` entry after you verified if this env variable works or not :)